### PR TITLE
Remove assertion requirements from 800-63B, since they are covered in…

### PR DESCRIPTION
**Pull request by jimfenton**
… 800-63C.

Avoided use of "assertion" in some contexts where distinction from -63C assertions may not be clear.

### CODE REVIEW

**Comment by jimfenton** [comment]
@jricher It looks like you just updated section 7.2.3, so I assumed it's still current but please double-check.

File: sp800-63b/sec7_session.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
 
-To facilitate this behavior, a *session* MAY be started in response to an authentication event, and such a session SHALL continue until such time that it is terminated. The session MAY be terminated for any number of reasons, including but not limited to an inactivity timeout, an explicit logout event, or other means. The session MAY be continued through a reauthentication event (described in section 7.2), wherein the user repeats some or all of the initial authentication event, thereby asserting their presence again.
+To facilitate this behavior, a *session* MAY be started in response to an authentication event, and such a session SHALL continue until such time that it is terminated. The session MAY be terminated for any number of reasons, including but not limited to an inactivity timeout, an explicit logout event, or other means. The session MAY be continued through a reauthentication event (described in section 7.2), wherein the user repeats some or all of the initial authentication event, thereby estblishing their presence again.
```
Comment (jricher) [inline sp800-63b/sec7_session.md]: Typo: "establishing"

**Review by jricher — CHANGES_REQUESTED** [review]
Needs one typo fixed but otherwise it's good.

**Comment by jricher** [comment]
Actually one more change: the table in 4.5 also needs to be updated to remove the assertion reference.

**Review by jricher — APPROVED** [review]

**Pull request merged**